packages/wrapper/src/languageClientWrapper.ts

```diff
@@ -23,10 +23,16 @@ export type LanguageClientConfig = {
     restartOptions?: LanguageClientRestartOptions;
 }
 
+export type LanguageClientRestartOptions = {
+    retries: number;
+    timeout: number;
+    keepWorker?: boolean;
+}
+
```

Comment:
what about using interfaces here as well?

Reply:
There are more places, where this applies. Give me a couple of minutes...